### PR TITLE
Fix for issue 2088 [ Wrong information about floating notation * ]

### PR DESCRIPTION
**Pull request by cristinamanum**
Doc updates for issue: https://github.com/NuGet/docs.microsoft.com-nuget/issues/2088

### CODE REVIEW

**Comment by anangaur** [comment]
Docs Build status updates of commit _[8e34d0f](https://github.com/NuGet/docs.microsoft.com-nuget/commits/8e34d0f67e09c219645d24fb9624d4191fc48fde)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/Package-Versioning.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-cmanu-2088/docs/concepts/Package-Versioning.md) | :bulb:Suggestion | [View](https://review.docs.microsoft.com/en-us/nuget/concepts/package-versioning?branch=pr-en-us-2132) | [Details](#user-content-9bd79866-d82f330e)

<a id="9bd79866-d82f330e"></a>
### [docs/concepts/Package-Versioning.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-cmanu-2088/docs/concepts/Package-Versioning.md)
 - **Line 34, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 57, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 226, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 232, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````
 - **Line 237, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/code-block-indented?branch=master)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning in late September, 2020.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C8%5C13%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202008132022130597-2132%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=6FftWcWOAQSp%2FGSGHfJeyGm35sSNPwm%2F92DBuzrsImM%3D&st=2020-08-13T20%3A17%3A45Z&se=2020-09-13T20%3A22%3A45Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: docs/concepts/Package-Versioning.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ When referring to package dependencies, NuGet supports using interval notation f
 | [1.0,2.0) | 1.0 ≤ x < 2.0 | Mixed inclusive minimum and exclusive maximum version |
 | (1.0)    | invalid | invalid |
 
-When using the PackageReference format, NuGet also supports using a floating notation, \*, for Major, Minor, Patch, and pre-release suffix parts of the number. Floating versions are not supported with the `packages.config` format.
+When using the PackageReference format, NuGet also supports using a floating notation, \*, for Major, Minor, Patch, and pre-release suffix parts of the number. Floating versions are not supported with the `packages.config` format. When a floating version is specified the rule is to resolve to the highest existent version that match the version description. Examples of loating versions and the resolutions are below.
```
Comment (Mikejo5000) [inline docs/concepts/Package-Versioning.md:112]:
```suggestion
When using the PackageReference format, NuGet also supports using a floating notation, \*, for Major, Minor, Patch, and pre-release suffix parts of the number. Floating versions are not supported with the `packages.config` format. When a floating version is specified, the rule is to resolve to the highest existent version that matches the version description. Examples of floating versions and the resolutions are below.
```

**Comment by ilya-spv** [comment]
Please also check the line that says:  and has 6.* as an example. It will not accept any, but only the highest

**Comment by anangaur** [comment]
Docs Build status updates of commit _[7902b29](https://github.com/NuGet/docs.microsoft.com-nuget/commits/7902b2932c7e3a578d0a5bb93c9cdc341607a77b)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/concepts/Package-Versioning.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-cmanu-2088/docs/concepts/Package-Versioning.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/nuget/concepts/package-versioning?branch=pr-en-us-2132) | [Details](#user-content-9bd79866-c983c499)

<a id="9bd79866-c983c499"></a>
### [docs/concepts/Package-Versioning.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/dev-cmanu-2088/docs/concepts/Package-Versioning.md)
 - **Line 5, Column 12**: **[Warning-[ms-author-invalid](https://review.docs.microsoft.com/en-us/help/contribute/validation-ref/ms-author-invalid?branch=main)]** `````Invalid value for 'ms.author', 'karann' is not a valid Microsoft alias.`````
 - **Line 34, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/author-not-found?branch=main)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning on 1/29/21.`````
 - **Line 57, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/author-not-found?branch=main)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning on 1/29/21.`````
 - **Line 226, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/author-not-found?branch=main)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning on 1/29/21.`````
 - **Line 232, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/author-not-found?branch=main)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning on 1/29/21.`````
 - **Line 237, Column 1**: **[Suggestion-[code-block-indented](https://review.docs.microsoft.com/help/contribute/validation-ref/author-not-found?branch=main)]** `````Indented code blocks aren't allowed. Use a Markdown code block surrounded by triple backticks (```). NOTE: This Suggestion will become a Warning on 1/29/21.`````

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C10%5C19%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202010192300593330-2132%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=h38p3kkHRwgsu4FpYqJFeKo3HdIrNEuci5zQvaKt%2BqU%3D&st=2020-10-19T22%3A56%3A35Z&se=2020-11-19T23%3A01%3A35Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://teams.microsoft.com/l/message/19:7ecffca1166a4a3986fed528cf0870ee@thread.skype/1590030602688?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&parentMessageId=1590030602688&teamName=Docs%20Support&channelName=General&createdTime=1590030602688) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/19a7edc0-f91d-0fd5-6e5f-4d131db05c11?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching in the <a href="https://review.docs.microsoft.com/en-us/search/?search=&amp;category=All&amp;scope=help-docs&amp;category=All&amp;branch=master">Docs contributor and Admin Guide</a></li><li>See the <a href="https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master">frequently asked questions</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Review by JonDouglas — APPROVED** [review]
LGTM!

**Pull request merged**